site/docs/numbers/searchingNumbers.mdx

```diff
@@ -17,261 +17,481 @@ export const accountId = "{accountId}";
-| NPA-NXX-X with Local Calling Area  | npaNxxx                 | rateCenter (state required), city (state required), state, lata, zip                              | quantity, LCA, enableTNDetail, protected     |
-| RateCenter                         | rateCenter, state       | city, areaCode/npaNxx/npaNxxx, lata, zip, orderBy                                                 | quantity, enableTNDetail, protected          |
-| RateCenter with Local Calling Area | rateCenter, state       |                                                                                                   | quantity, LCA, enableTNDetail, protected     |
-| State                              | state                   | rateCenter, city, areaCode/npaNxx/npaNxxx, lata, zip                                              | quantity, enableTNDetail, protected          |
-| City                               | state, city             | rateCenter, state, areaCode/npaNxx/npaNxxx, lata, zip, orderBy                                    | quantity, enableTNDetail, protected          |
-| Zip Code                           | zip                     | rateCenter (state required), city (state required), state, areaCode/npaNxx/npaNxxx, lata, orderBy | quantity, enableTNDetail, protected          |
-| LATA                               | lata                    | rateCenter (state required), city (state required), state, areaCode/npaNxx/npaNxxx, zip           | quantity, enableTNDetail, protected          |
-| Local Vanity                       | localVanity             | state, areaCode                                                                                   | endsIn, quantity, protected, enableTNdetails |
-| TollFree Vanity                    | tollFreeVanity          | orderBy                                                                                           | quantity                                     |
-| TollFree WildCard                  | tollFreeWildCardPattern | orderBy                                                                                           | quantity                                     |
-
-### Examples
+To make a search for the available number, you must make a GET request to our [API available numbers](../../../../apis/numbers#tag/accounts/operation/GetAvailableTns) endpoint.
+This can be done through our [SDKs](/sdks), using tools like [Postman](https://github.com/Bandwidth/postman) or in the command line.
+
+:::tip Searching Toll Free Numbers
+The `tollFreeWildCardPattern` allows you to search toll free area codes, but forces the 3rd digit to be a `*`.
+However, with the nature of Toll Free area codes - the 3rd digit will always be a repeat of the second, so to search for 833 numbers, set the parameter value as `83*` and you will only return 833 numbers.
+For 800 numbers, search for `80*` and so on.
+:::
 
 <Tabs
```

Comment:
Is there a reason for replacing the different number search types with sdk examples? 

We dont really support the sdks and below is missing python and ruby